packages/models/src/plugins/client/client-hook.model.ts

```diff
@@ -97,7 +99,22 @@ export const clientFilterHookObject = {
   'filter:internal.player.videojs.options.result': true,
 
   // Filter p2p media loader options built for PeerTube player
-  'filter:internal.player.p2p-media-loader.options.result': true
+  'filter:internal.player.p2p-media-loader.options.result': true,
+
+  // Filter bulk actions in user list
+  'filter:admin-user-list.bulk-actions.create.result': true,
```

Comment:
`filter:admin-user-list.bulk-actions.create.result` for consistency (`s` to `user`). Please also do the same with `video-comment` -> `video-comments`